src/components/primitive/drop-down-menu/dropdown-menu.component.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { Avatar, Box } from 'components/primitive';
+import { IconContainer, DropDownContainer } from './dropdown-menu.component.styles';
+
+export const DropDownMenu = (props: any) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const { icon, children } = props;
+
+  return (
+    <Box>
+      <Box>
+        <IconContainer onClick={() => setIsActive(!isActive)}>
```

Comment:
We should make the drop down container inactive when clicked elsewhere on the screen too.